sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java

```diff
@@ -153,17 +152,25 @@ public void beginPurchaseandReleasePhoneNumbers(HttpClient httpClient) {
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
     public void beginUpdatePhoneNumberCapabilities(HttpClient httpClient) {
         String phoneNumber = getTestPhoneNumber(PHONE_NUMBER);
+
         StepVerifier.create(
-            beginUpdatePhoneNumberCapabilitiesHelper(httpClient, phoneNumber, "beginUpdatePhoneNumberCapabilities").last()
-            .flatMap((AsyncPollResponse<PhoneNumberOperation, PurchasedPhoneNumber> result) -> {
-                assertEquals(LongRunningOperationStatus.SUCCESSFULLY_COMPLETED, result.getStatus());
-                return result.getFinalResult();
-            })
+            this.getClientWithConnectionString(httpClient, "getPurchasedPhoneNumberForCapabilities").getPurchasedPhoneNumberWithResponse(phoneNumber)
+                .flatMap(responseAcquiredPhone -> {
+                    PhoneNumberCapabilities oldPhoneNumberCap = responseAcquiredPhone.getValue().getCapabilities();
+                    PhoneNumberCapabilities newPhoneNumberCap = getNewPhoneCapabilities(oldPhoneNumberCap);
```

Comment:
Can we move the declaration of these two variables outside "this.getClientWithConnectionString"? If yes, we can assert the update actually caused capability change, assertNotEqual.

Reply:
Cant because first need to check the current state of the phoneCapabilities